.vscode/settings.json

```diff
@@ -1,7 +1,7 @@
 {
 	"eslint.codeActionsOnSave.rules": ["curly"],
 	"editor.codeActionsOnSave": {
-		"source.fixAll.eslint": true
+		"source.fixAll.eslint": "explicit"
```

Comment:
I think this might have been accidentally committed?

Reply:
@BrianMMcClain I honestly don't know what this is....I assume it should be `true` and not `explicit`?

Reply:
@eganhashicorp I think this is a settings file from VS Code, mine made the same change when I opened up the changes locally. My two cents would be to swap it back to true because I'm not sure what breaks if it changes 😁